README.adoc

```diff
@@ -157,18 +157,18 @@ have IPv4 addresses, the address on eth2 is used.
 === HTTP Proxy Settings
 
 In an environment where the HTTP traffic needs to pass through an
-HTTP proxy server, set the `http_proxy`, `http_proxy_user` and
-`http_proxy_password` proxy configurations in the Vagrantfiles to enable
+HTTP proxy server, set the `proxy`, `proxy_user` and
```

Comment:
Ideally we should not change user facing variables of config names and communicate if we do something like that as @hferentschik  mentioned. However in this case we should be ok to change it as it was not working at all before and we are trying the fix the issue.